### PR TITLE
feat: add copilot check treeview node

### PR DESCRIPTION
**Pull request by swatDong**
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/24918417/

- Controlled by COPILOT_PLUGIN feature flag
- Show copilot status together with sideloading

E.g.,
image
image
image

### CODE REVIEW

File: packages/vscode-extension/package.nls.json [hunk 1]
```diff
@@ -1,5 +1,13 @@
...
+    "teamstoolkit.accountTree.copilotEnroll": "Enroll in Early Access Program",
+    "teamstoolkit.accountTree.copilotMessage": "Microsoft 365 account admin haven't enabled Copilot access for this account. Contact your Teams administrator to resolve this issue by enrolling in Microsoft 365 Copilot Early Access program.",
+    "teamstoolkit.accountTree.copilotPass": "Copilot Access Enabled",
+    "teamstoolkit.accountTree.copilotPassTooltip": "You have Copilot access already.",
+    "teamstoolkit.accountTree.copilotStatusUnknown": "Copilot Access Check Failed",
+    "teamstoolkit.accountTree.copilotStatusUnknownTooltip": "The Copilot access of this account is unknown due to network or service issue. Please refresh again later to check it.",
+    "teamstoolkit.accountTree.copilotWarning": "Copilot Access Disabled",
+    "teamstoolkit.accountTree.copilotWarningTooltip": "Microsoft 365 account admin haven't enabled Copilot access for this account.\nContact your Teams administrator to resolve this issue by enrolling in Microsoft 365 Copilot Early Access program.\nVisit: https://aka.ms/PluginsEarlyAccess",
```
Comment (HuihuiWu-Microsoft) [inline packages/vscode-extension/package.nls.json:10]: Better extract the link from the string for better localization practice.

Comment (swatDong) [reply packages/vscode-extension/package.nls.json:10]: This string is referenced in `package.json` as command title. https://github.com/OfficeDev/TeamsFx/blob/qidon/copilotcheck/packages/vscode-extension/package.json#L748

Does that support extraction as well?

Comment (HuihuiWu-Microsoft) [reply packages/vscode-extension/package.nls.json:10]: OK, then no need to extract~

**Review by HuihuiWu-Microsoft — APPROVED** [review]

**Review by kuojianlu — DISMISSED** [review]

File: packages/vscode-extension/package.nls.json (hunk 1 above)
Comment (sffamily) [inline packages/vscode-extension/package.nls.json:10]: same comment as the similar line above.

File: packages/vscode-extension/package.nls.json [hunk 2]
```diff
@@ -1,5 +1,13 @@
 {
     "teamstoolkit.accountTree.azureAccountTooltip": "AZURE ACCOUNT  \nThe Teams Toolkit may require an Azure subscription to deploy the Azure resources for your project.",
+    "teamstoolkit.accountTree.copilotEnroll": "Enroll in Early Access Program",
```
[the remark this answers is not in the document]
Comment (swatDong) [reply packages/vscode-extension/package.nls.json]: Updated

File: packages/vscode-extension/package.nls.json [hunk 3]
```diff
@@ -1,5 +1,13 @@
...
+    "teamstoolkit.accountTree.copilotMessage": "Microsoft 365 account admin haven't enabled Copilot access for this account. Contact your Teams administrator to resolve this issue by enrolling in Microsoft 365 Copilot Early Access program.",
```
[the remark this answers is not in the document]
Comment (swatDong) [reply packages/vscode-extension/package.nls.json]: Updated to `administrator`.

**Review by kuojianlu — APPROVED** [review]

**Review by sffamily — APPROVED** [review]

**Review by HuihuiWu-Microsoft — APPROVED** [review]

**Review by jayzhang — APPROVED** [review]

**Pull request merged**